pallets/parachain-staking/src/tests.rs

```diff
@@ -3597,9 +3641,13 @@ fn nominations_merged_before_reward_payout() {
 				Event::CollatorChosen(2, 4, 50),
 				Event::NewRound(5, 2, 4, 200),
 				Event::Rewarded(3, 1),
+				Event::NominatorDueReward(5, 3, 1),
 				Event::Rewarded(4, 1),
+				Event::NominatorDueReward(5, 4, 1),
 				Event::Rewarded(1, 1),
+				Event::NominatorDueReward(5, 1, 1),
 				Event::Rewarded(2, 1),
+				Event::NominatorDueReward(5, 2, 1),
 				// ALL REWARDS FOR 5 are merged into one payment + event
 				Event::Rewarded(5, 4),
```

Comment:
This is why the event is useful for nominators. Without the event, they don't see exactly where the rewards come from...